styles/styles.css

```
@@ -1,7 +1,284 @@
+}
+
+/* ==== GRIDS (SECOND SECTION) ==== */
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(3, 1fr);
+  gap: 20px;
+  margin: 20px 0;
+}
+
+/* Title for Best Selling Products */
+
+.title-products {
+  font-weight: 600;
+  /* ==== COMMON ==== */
+  font-size: var(--title-poppins);
+}
+
+
+/* Product Box */
+.product-box {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 10px;
+  /* ==== COMMON ==== */
+  background-color: var(--color-white);
+  box-shadow: 0 4px 8px var(--color-grey);
+  padding: 20px 0 10px 0;
+  position: relative;
+}
+
+/* Badge at the top of the product box */
+.discount-badge {
+  /* ==== COMMON ==== */
+  background-color: var(--color-primary-red);
+  color: var(--color-white);
+  padding: 5px 10px;
+  border-radius: 5px;
+  position: absolute;
+  top: 0;
+  left: 0;
+}
+
+.product-box img {
+  max-width: 100%;
+  margin-top: 20px;
+  height: 170px;
+}
+
+/* Text in the Product Box */
+.home-page .text-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin: 10px 0 0 40px;
+  width: 100%;
+}
+
+.home-page .text-container h3 {
+  /* ==== COMMON ==== */
+  font-size: var(--h3-poppins);
+  margin: 0;
+  width: max-content;
+}
+
+.price-container {
+  display: flex;
+  flex-direction: row;
+  gap: 10px;
+  margin: 0;
+}
+
+.price-container p {
+  margin: 0;
+}
+
+.price-normal {
+  text-decoration: line-through;
+  /* ==== COMMON ==== */
+  color: var(--color-grey);
+}
+
+.price-discount {
+  /* ==== COMMON ==== */
+  color: var(--color-primary-green);
+  font-weight: bold;
+}
+
+.text-container button {
+  /* ==== COMMON ==== */
+  background-color: var(--color-primary-red);
+  color: var(--color-white);
+  padding: 10px 20px;
+  border: none;
+  border-radius: 5px;
+  /* ==== COMMON ==== */
+  font-size: var(--button-cart);
   font-family: var(--font-primary);
+  margin: 10px 0;
+  width: fit-content;
+  cursor: pointer;
+  transition: background-color 0.3s ease, transform 0.2s ease;
+}
+
+.text-container button:hover {
+  /* ==== COMMON ==== */
+  background-color: var(--color-secondary-light-red);
+  transform: scale(1.05);
+}
+
+/* ==== BANNER (THIRD SECTION) ==== */
+.banner-style {
+  /* ==== COMMON ==== */
+  background-color: var(--color-neutral-black);
+  width: 100%;
+  height: 300px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 20px;
+  box-sizing: border-box;
+}
+
+/* Subtitle at the top of the banner */
+.banner-style .poppins-style {
+  /* ==== COMMON ==== */
+  color: var(--color-white);
+  font-weight: 300;
+  /* ==== COMMON ==== */
+  font-size: var(--subtitle-banner);
+  letter-spacing: 0.3em;
+  margin: 10px;
+}
+
+/* Container for text, icons, and image */
+.home-page .text-img-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-around;
+  width: 100%;
+}
```

Esto hara que se vea como columna en celulares y en desktop como fila 

```suggestion
.home-page .text-img-container {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: space-around;
  width: 100%;
}

@media (min-width: 576px) {
  .home-page .text-img-container {
    flex-direction: row;
  }
}
```

También podrías usar la clase `flex-responsive-row` que había creado